server-connect-base/README.md

```diff
@@ -0,0 +1,38 @@
+# Confluent Server Docker Image for Kafka Connect
+
+Docker image for deploying and running Kafka Connect. 
+
+_Functionally, the `cp-server-connect` and the `cp-server-connect-base` images are identical. Prior to Confluent Platform 6.0 the `cp-server-connect` image included several connectors pre-installed, but **this is no longer the case**._
```

Comment:
Some are still pre-installed, I think Replicator and maybe FileStream?  Please check me on this.

Reply:
FileStream is part of Apache Kafka so is included. Replicator is not included.